### DISCUSSION TITLE
How mature is the syntax of V0.4.4?

### DISCUSSION THREAD

**Discussion by Jengro777** [vlang/v#20871] [Questions and Answers] [answered]
Whether we wait for V0.5 or can use Vlang for production now depends on the maturity of Vlang's syntax. After all, no one wants to update to V1.0 in the future and need to modify a lot of code to get the project running.

**Comment by JalonSolov** [comment, marked as answer]
Using code for production has nothing to do with changes that might be needed in the future.  If you need to go to production now, you go with what you have.

**Comment by medvednikov** [comment]
V has had few changes in syntax for a pre-1.0 language. Most of the changes are handled by vfmt (like the old go => spawn).

I'd say it's pretty mature right now.

I can't think of any major changes planned before 1.0.
